frontend/redux/UserReducer.tsx

```diff
@@ -0,0 +1,22 @@
+import { combineReducers } from "redux";
+
+const INITIAL_STATE = {
+  _id: "",
```

Comment:
I don't think it's worthwhile to story `_id` or even to return it from the database. We don't really use it. We should probably store the Admin object here rather than all the fields of the Admin object.